languages/elixir/exercises/concept/binary-matching/.meta/design.md

```diff
@@ -0,0 +1,40 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student about the binary data type in Elixir
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know what binary data is
+- Know that strings are a form of encoded binary data
+- Know what the `<<>>` special form is
+- Know that each value refers to a byte
+- Know that can modify the size of a value in a `<<>>` form using `::`
```

Comment:
Is this covered in the example file? If not, it can be totally okay to just have it mentioned in the introduction/after document, but just wanted to mention it.